src/main/java/org/apereo/openlrs/storage/elasticsearch/NormalizedElasticsearchTierTwoStorage.java

```diff
@@ -49,6 +49,10 @@ public OpenLRSEntity findById(String id) {
 	}
 
 	@Override
+	public OpenLRSEntity findBySourceId(String sourceId) { throw new UnsupportedOperationException(); }
+
+
+		@Override
```

Comment:
Yes.  As you know, I like code to be well-formatted.  This application's code has a few different styles of indentation.  Some files use 2-space rather than 4-space indents.  I think other files use tabs.  There might be mixtures of the styles within a file, too.  I'm hesitant to impose my preference on the code, so I'm trying to fit in with the existing styles.

I should ask @gryglbrt what he prefers.